lib/error-helper.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const getStandardErrorResponse = function(err, res) {
```

Comment:
What do you think about tweaking this?

```js
function checkResponseStatus(badgeData, err, res) {
    if (err != null) {
        badgeData.text[1] = 'inaccessible';
        badgeData.colorscheme = 'red';
        return true;
    }
    ...
    return false;
}
```

Then it could be invoked more tersely:

```js
if (checkResponseStatus(badgeData, err, res)) {
    sendBadge(format, badgeData);
    return;
}
```

This version is slightly harder to test, but seems worth it.